### ISSUE TITLE
Installing alabaster-0.7.5 raises UnicodeDecodeError

### ISSUE THREAD

**Issue by nre** [rtfd/readthedocs.org#1350]
alabaster-0.7.5 was released 2015-06-15 and is breaking the installation of Sphinx.
NB: pip can install this new version on my machine without errors.

https://readthedocs.org/builds/doxhooks/2969595/
```
doc_builder
-----
[...]
Collecting alabaster<0.8,>=0.7 (from sphinx==1.3.1)
  Using cached alabaster-0.7.5.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-qgmxgtcw/alabaster/setup.py", line 13, in <module>
        readme = f.read()
      File "/home/docs/checkouts/readthedocs.org/user_builds/doxhooks/envs/latest/lib/python3.4/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 15684: ordinal not in range(128)
```

**Comment by dan-blanchard** [comment]
I'm also seeing this issue.

**Comment by ericholscher** [comment]
I don't see the issue with our side? It seems this is an alabaster issue?
Have you raised it with them?

On Tue, Jun 16, 2015 at 10:26 PM, Dan Blanchard <notifications@github.com>
wrote:

> I'm also seeing this issue.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rtfd/readthedocs.org/issues/1350#issuecomment-112625836>
> .
>



-- 
Eric Holscher
Maker of the internet residing in Portland, Oregon
http://ericholscher.com

**Comment by dan-blanchard** [comment]
It seems like it's only with Python 3.  They have an issue open about it bitprophet/alabaster#40.

**Comment by agjohnson** [comment]
Closing this here, as this is an alabaster issue.

**Comment by nre** [comment]
Why wait for a third party to sort themselves out? You can prevent the broken version of alabaster being installed. Before installing Sphinx, tell pip to avoid alabaster 0.7.5:

    pip install alabaster!=0.7.5

**Comment by agjohnson** [comment]
I'm -1 on this. It may not work for your case, but others will wonder why 0.7.5 doesn't install. Seems easy enough for the doc maintainer to handle anyways.

**Comment by flacjacket** [comment]
I don't see how to set the version of alabaster pulled in for the doc builder, trying to add `alabaster!=0.7.5` to a requirements.txt still pulls in 0.7.5 when sphinx is installed, failing the build. While this is because of an alabaster bug, all Python 3 RTD builds will fail with 0.7.5.

**Comment by agjohnson** [comment]
We are going to put in a patch to resolve this, rather than waiting for the fix in 0.7.6. We don't officially support Alabaster, and keeping up to date with third party, user-installable package versions is something we'd like to avoid due to maintenance concerns. But seems we should take a stance on what we consider officially supported, and having procedure around keeping those current and stable.
